### PR TITLE
Update organization-exports enums

### PR DESCRIPTION
**Pull request by nanobash**

### CODE REVIEW

File: openapi/components/schemas/OrganizationExports/OrganizationExportResource.yaml [hunk 1]
```diff
@@ -13,10 +13,14 @@ properties:
...
-      - webhooks
-      - webhooks-tracking
       - blocklists
       - lists
+      - webhooks
+      - products
+      - websites
+      - plans
+      - credit-memos
+      - email-notifications
```
Comment (VladimirBerdnik) [inline openapi/components/schemas/OrganizationExports/OrganizationExportResource.yaml:23]: files?

File: openapi/components/schemas/OrganizationExports/OrganizationExportResource.yaml [hunk 2]
```diff
@@ -26,10 +30,14 @@ properties:
...
-       webhooks: All webhooks of the organization.
-       webhooks-tracking: All webhooks tracking items of the organization.
        blocklists: All blocklists of the organization.
        lists: All lists of the organization.
+       webhooks: All webhooks of the organization.
+       products: All products of the organization.
+       websites: All websites of the organization.
+       plans: All the plans of the organization.
+       credit-memos: All the credit-memos of the organization.
+       email-notifications: All the email-notifications of the organization.
```
Comment (VladimirBerdnik) [inline openapi/components/schemas/OrganizationExports/OrganizationExportResource.yaml:40]: `All products` but `All the plans`. I think we must use one style

File: openapi/components/schemas/OrganizationExports/OrganizationExportResource.yaml (hunk 1 above)
Comment (nanobash) [reply openapi/components/schemas/OrganizationExports/OrganizationExportResource.yaml:23]: Files not implemented yet. I will do separate PR for files. These updates were missing on api side

File: openapi/components/schemas/OrganizationExports/OrganizationExportResource.yaml (hunk 2 above)
Comment (nanobash) [reply openapi/components/schemas/OrganizationExports/OrganizationExportResource.yaml:40]: What do you mean?

**Review by VladimirBerdnik — APPROVED** [review]

**Review by p3yman — APPROVED** [review]

**Pull request merged**